src/main.rs

```diff
@@ -1,11 +1,18 @@
 use anyhow::Context;
 use snmp_sim::app::Service;
 use snmp_sim::configuration::get_configuration;
+use clap::Parser;
```

Comment:
- move this to ./src/cli/cli.rs
The CLI parser implementation will not be part this module. The main.rs will only call the appropriate function to parse the CLI.